apps/bbsync/tests/test_query.py

```diff
@@ -208,6 +208,26 @@ def test_generate_component(self, is_draft, result):
         bbq = FlawBugzillaQueryBuilder(flaw)
         assert bbq.query["component"] == result
 
+    @pytest.mark.parametrize("has_cve", [True, False])
+    def test_generate_alias(self, has_cve):
+        """
+        test generating of alias on creation
+        """
+        alias_id = "CVE-2000-1001" if has_cve else "GHSA-0001"
+
+        if has_cve:
+            flaw = FlawFactory(cve_id=alias_id)
+        else:
+            snippet = SnippetFactory(
+                source=Snippet.Source.OSV, ext_id=alias_id, cve_id=None
+            )
+            flaw = FlawFactory(cve_id=None, meta_attr={"external_ids": [alias_id]})
+            snippet.flaw = flaw
+            snippet.save()
+
+        bbq = FlawBugzillaQueryBuilder(flaw)
+        assert bbq.query["alias"] == [alias_id]
```

Comment:
This test has only two lines of the join code between the parametrized cases and the parametrization adds two extra lines, branching, and conditions. I do not see it beneficial. Having two separate but much simpler test cases would be IMO much better.